cli/login.go

```diff
@@ -398,6 +460,48 @@ func (r *RootCmd) login() *serpent.Command {
 			Description: "By default, the CLI will generate a new session token when logging in. This flag will instead use the provided token as the session token.",
 			Value:       serpent.BoolOf(&useTokenForSession),
 		},
+		{
+			Flag:        "first-user-first-name",
+			Env:         "CODER_FIRST_USER_FIRST_NAME",
+			Description: "Specifies the first name of the user.",
+			Value:       serpent.StringOf(&firstName),
+		},
+		{
+			Flag:        "first-user-last-name",
+			Env:         "CODER_FIRST_USER_LAST_NAME",
+			Description: "Specifies the last name of the user.",
+			Value:       serpent.StringOf(&lastName),
+		},
+		{
+			Flag:        "first-user-phone-number",
+			Env:         "CODER_FIRST_USER_PHONE_NUMBER",
+			Description: "Specifies the phone number of the user.",
+			Value:       serpent.StringOf(&phoneNumber),
+		},
+		{
+			Flag:        "first-user-job-title",
+			Env:         "CODER_FIRST_USER_JOB_TITLE",
+			Description: "Specifies the job title of the user.",
+			Value:       serpent.StringOf(&jobTitle),
+		},
+		{
+			Flag:        "first-user-company-name",
+			Env:         "CODER_FIRST_USER_COMPANY_NAME",
+			Description: "Specifies the company name of the user.",
+			Value:       serpent.StringOf(&companyName),
+		},
+		{
+			Flag:        "first-user-country",
+			Env:         "CODER_FIRST_USER_COUNTRY",
+			Description: "Specifies the country of the user.",
+			Value:       serpent.StringOf(&country),
+		},
+		{
+			Flag:        "first-user-developers",
+			Env:         "CODER_FIRST_USER_DEVELOPERS",
+			Description: "Specifies the number of developers.",
+			Value:       serpent.StringOf(&developers),
+		},
```

Comment:
I'd rather not expose these as CLI arguments.

Reply:
I agree with Cian here.

Reply:
got it, will remove these